### ISSUE TITLE
Alternative API Server ports require manual patching

### ISSUE THREAD

**Issue by andrewwebber** [kubernetes-incubator/bootkube#613]
Considering the following command...
```bash
bootkube render --asset-dir=assets --api-servers=https://192.168.122.10:1443 --api-server-alt-names=DNS=k8.example.com,IP=192.168.122.10 --etcd-servers=https://192.168.122.10:2379
```

bootkube process does not set the secure port on the api server to be 1443. This can catch some user out as the kubelet never finds the api server and all other manifest containers get launched.

My main use case for launching the api server on an alternative port was to free up ingress containers to own port 80 and 443

**Comment by diegs** [comment]
Yes, this is a bug, fix is in #597 (but requires a Go version bump).

**Comment by diegs** [comment]
Fixed with #597
